allensdk/brain_observatory/dataframe_utils.py

```diff
@@ -0,0 +1,87 @@
+from typing import List
+import pandas as pd
+
+
+def patch_df_from_other(
```

Comment:
I would argue that this should be in a more central location, rather than brain_observatory

Reply:
moved to allensdk.core